admin-core/src/main/kotlin/com/epam/drill/admin/DrillApplication.kt

```diff
@@ -99,10 +99,10 @@ fun Application.module() {
         loginRoute()
         route("/api") {
             userAuthenticationRoutes()
-            authenticate("jwt") {
+            authenticate("session", "jwt") {
                 userProfileRoutes()
             }
-            authenticate("jwt", "basic") {
+            authenticate("session", "jwt", "basic") {
```

Comment:
1. _tbd_ I don't think we need "basic" for `userManagementRoutes`, am I missing smth?
I thought we'd use `"basic"` only for non-browser clients (CI/CD scripts, our components) -> so it should be set only for drill HTTP API
2. is `"oauth"` missing for other endpoints in drill http api (ctrl+shift+f for `authenticate(` string)